recipes/gfan/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export CPPFLAGS="-I$PREFIX/include -DDISABLE_COMMENTATOR $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib -lcddgmp -lgmp -lm $LDFLAGS"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CFLAGS="-O2 -g -fPIC $CFLAGS"
+export CXXFLAGS="-O2 -g -fPIC $CXXFLAGS"
+
+make
+mkdir -p "$PREFIX/bin"
+cp -pf gfan "$PREFIX/bin/"
+cd "$PREFIX/bin"
+./gfan installlinks
```

Comment:
Yes, I'll add a comment in the feedstock. This creates symlinks to the same executable. The symlinks are shortcuts to some functionality which can be invoked from the main executable as well.